rescue/2025-09-16/cosmic-helix/helix-renderer.mjs

```
@@ -0,0 +1,497 @@
+}
+
+// Layer 3: Fibonacci curve — golden spiral polyline anchored to centre.
+function drawFibonacci(ctx, w, h, color, N) {
+  ctx.save();
+  ctx.strokeStyle = color;
+  ctx.lineWidth = 2;
+  ctx.lineCap = "round";
+
+  const fib = buildFibonacci(N.NINE);
+  const scale = Math.min(w, h) / N.ONEFORTYFOUR;
+  const fib = [1, 1];
```

Biome (2.1.2)

[error] Shouldn't redeclare 'fib'. Consider to delete it or rename it.

'fib' is defined here:


(lint/suspicious/noRedeclare)